chat.tox.qTox.json

```diff
@@ -0,0 +1,44 @@
+{
+	"app-id": "chat.tox.qTox",
+	"runtime": "org.kde.Platform",
+	"sdk": "org.kde.Sdk",
+	"runtime-version": "5.10",
+	"command": "qtox",
+	"finish-args": [
+		"--share=network",
+		"--socket=pulseaudio",
+		"--socket=wayland",
+		"--socket=x11",
+		"--filesystem=host",
```

Comment:
what do you need the host filesystem for?

Reply:
I think the following filesystem permissions would be enough for most people?

"--filesystem=home", -> Be able to save received files in your home where ever you want.
"--filesystem=/media:ro", -> Be able to send something from a mounted USB stick.
"--filesystem=xdg-run/gvfs:ro", -> Be able to send something from a for example mtp:// device which was automatically mounted by gvfs-fuse

Reply:
If you only need users to select files to upload and select locations to download you need **no** file system access. Qt supports the Flatpak portal and works out of the box.

You only need filesystem access if you access files outside of that usage.

Reply:
If I don't set this filesystem flags, I can not send files with qTox nor can I save received files?

Reply:
Sure you can. You use a normal Qt file chooser dialog, the user selects the path, and then it returns a path you can read/write.